.github/workflows/update-drci-comments.yml

```diff
@@ -0,0 +1,22 @@
+name: Update Dr. CI comments
+
+on:
+  schedule:
+    # Run every 15 minutes
+    - cron: "*/15 * * * *"
+  # Have the ability to trigger this job manually through the API (temporary)
+  workflow_dispatch:  
+
+jobs:
+  update-drci-comments:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Retrieve rockset query results and update Dr. CI comments
+        run: |
```

Comment:
Can't we just make it an  .mjs script and call it here? I don't think this needs to be in an API since it just adds extra an extra dependency on the server and additional load as well.

Reply:
Also it should solve the problem of passing in a auth key.